config/navier_stokes_config.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any, List, Optional
+
+from zencfg import ConfigBase
+from .distributed import DistributedConfig
+from .datasets import NavierStokesDatasetConfig
+from .models import ModelConfig, SimpleFNOConfig
+from .opt import OptimizationConfig, PatchingConfig
+from .wandb import WandbConfig
+
+class Default(ConfigBase):
+    n_params_baseline: Optional[Any] = None
+    verbose: bool = True
+    distributed: DistributedConfig = DistributedConfig()
+    model: ModelConfig = SimpleFNOConfig(
+        data_channels=1,
+        out_channels=1,
+        n_modes=[64,64],
+        hidden_channels=64,
+        n_layers=4,
+        projection_channel_ratio=4,
+    )
+    opt: OptimizationConfig = OptimizationConfig(
+        n_epochs=600,
+        learning_rate=3e-4,
+        training_loss="h1",
+        weight_decay=1e-4,
+        scheduler="StepLR",
+        step_size=100,
+        gamma=0.5,
+    )
+    data: NavierStokesDatasetConfig = NavierStokesDatasetConfig(
```

Comment:
Let's put all the values for NS in the main one I think, we don't need to override it here. If you want to be able to inspect, just put the whole NavierStokesDatasetConfig declaration in this file.